staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -70,7 +72,16 @@ func (v *Error) ErrorBody() string {
 			if reflectValue := reflect.ValueOf(value); reflectValue.IsNil() {
 				value = "null"
 			} else {
-				value = reflectValue.Elem().Interface()
+				// Do not use the pointer value if the pointer implements fmt.Stringer
+				if !reflectValue.Type().Implements(stringerType) {
+					value = reflectValue.Elem().Interface()
+				}
+			}
+		} else {
+			valuePtr := reflect.New(valueType)
+			if valuePtr.Type().Implements(stringerType) {
+				valuePtr.Elem().Set(reflect.ValueOf(value))
+				value = valuePtr.Interface()
```

Comment:
yeah, I wouldn't add this... this seems too intricate

Reply:
done